### PR TITLE
Testsuite reorganization, new feature to call exported functions

### PR DESCRIPTION
**Pull request by turleypol**
Testsuite is now pkg based
Added os::LoadExportedScript and new object ExportedScript to call exported functions from a different script.

### CODE REVIEW

File: pol-core/doc/core-changes.txt [hunk 1]
```diff
@@ -1,7 +1,16 @@
 ﻿-- POL100 --
+08-20-2020 Turley:
+    Added: os::LoadExportedScript(scriptname, args:={})
+           Load and start program part of given script. The optional args array will be unpacked and passed as single arguments to the program. Script will be started non-critical or critical depending on the calling state. Blocks until the program part ends. Returns array{ExportScript, return value} return value is the return value of the program or 1 if nothing was returned.
+    Added: ExportScript object
+           the script will remain loaded as long as the object exists.
+           member: .exported_functions array of exported function names
+           method .call(functionname, args:={})
+           calls and blocks current scriot a exported function and returns the return value or 1 if no return value was given. Like LoadExportedScript critical state depends on the calling script. Args see LoadExportedScript.
```
Comment (eric-swanson) [inline pol-core/doc/core-changes.txt:9]: typo `scriot`

**Review by KevinEady — COMMENTED** [review]
just some questions

File: pol-core/pol/module/osmod.cpp [hunk 2]
```diff
@@ -814,6 +812,8 @@ void OSExecutorModule::SleepForMs( int msecs )
     blocked_ = true;
     wait_type = Core::WAIT_TYPE::WAIT_SLEEP;
     sleep_until_clock_ = Core::polclock() + msecs * Core::POLCLOCKS_PER_SEC / 1000;
+    if ( !sleep_until_clock_ )
+      sleep_until_clock_ = 1;
```
[the remark this answers is not in the document]
Comment (turleypol) [reply pol-core/pol/module/osmod.cpp:816]: It's 0 directly after a fresh start of the Testsuite. (If in pol.txt is no clock stored)

[the remark this answers is not in the document]
Comment (turleypol) [reply pol-core/pol/module/osmod.cpp:816]: Yes negatives are in theory allowed, that should be also changed true

**Pull request merged**